.github/workflows/ci.yml

```diff
@@ -0,0 +1,111 @@
+name: Continuous Integration
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    tags:
+      - '*'
+    branches:
+      - main
+      - '*.x'
+
+jobs:
+  ci:
+    name: ci
+    strategy:
+      matrix:
+        system: ["ubuntu-20.04"]
+        jvm: [8]
+        scala: ["2.13.11"]
+        espresso: ["2.4"]
+        circt: ["firtool-1.48.0"]
+    uses: ./.github/workflows/test.yml
+    with:
+      system: ${{ matrix.system }}
+      jvm: ${{ matrix.jvm }}
+      scala: ${{ matrix.scala }}
+      espresso: ${{ matrix.espresso }}
+      circt: ${{ matrix.circt }}
+
+  # Sentinel job to simplify how we specify which checks need to pass in branch
+  # protection and in Mergify. This job checks that all matrix jobs were
+  # successful.
+  check-tests:
+    name: "check tests"
+    needs: ci
+    runs-on: ubuntu-20.04
```

Comment:
Note: I removed the `if: success()` part of this job. According to the GitHub documentation, this is the default option.